libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -536,6 +536,20 @@ bool GLSLPostProcessor::fullOptimization(const TShader& tShader,
             glslOptions.emit_uniform_buffer_as_plain_uniforms = true;
         }
 
+        if (config.variant.hasStereo() && config.shaderType == ShaderStage::VERTEX) {
+            switch (config.materialInfo->stereoscopicType) {
+            case StereoscopicType::INSTANCED:
+                // Nothing to generate
+                break;
+            case StereoscopicType::MULTIVIEW:
+                // For stereo variants using multiview feature, this generates the shader code below.
+                //   #extension GL_OVR_multiview2 : require
+                //   layout(num_views = 2) in;
+                glslOptions.ovr_multiview_view_count = config.materialInfo->stereoscopicEyeCount;
```

Comment:
Is this meant for the backend to consume?  If so, how will you keep it in sync with the spec constant value?  I wonder how the metal backend is using the spec constant.

Reply:
This is for spirv-cross to generate glsl code when we pass  the optimize (-S) param or target Metal/Vulkan to matc. Normally this pass will be not hit unless we generate glsl code using matinfo. The spec constant is updated with the same value.